linter_plugin.py

```diff
@@ -10,7 +10,7 @@
 from pylint.interfaces import IAstroidChecker
 
 # Modules in theses packages can import the os module.
-WHITELIST_PACKAGES = ['acme', 'certbot_compatibility_test', 'lock_test']
+WHITELIST_PACKAGES = ['acme', 'certbot-ci', 'certbot_compatibility_test', 'lock_test']
```

Comment:
The root cause of why `# pylint: disable=os-module-forbidden` is still need in integrations tests is here: in fact it is the module name that needs to be referenced here, not the folders where they are. So `certbot-ci` should be replaced with certbot_integration_tests`.

Reply:
`certbot-ci/certbot_integration_tests/utils/dns_server.py:112:0: W0311: Bad indentation. Found 8 spaces, expected 12 (bad-indentation)` This is weird.

Reply:
My local branch is rebased against certbot/master. All of these warnings are shown, when doing the replacement.

Reply:
@atombrella since I wrote the new code in `certbot-ci` which falls afoul of your pylint and mypy changes, here is [a patch](https://gist.github.com/alexzorin/e4a0dddfcd20d60f949661a3d7d11059) which will address the warnings. 

You will need to rebase on with master again, before applying it.

Reply:
@alexzorin Could you please consider enabling/installing a plugin for https://editorconfig.org/ in your editor? The indentation mess should be prevented with this.

Reply:
@alexzorin Your patch does not apply :/